src/main/java/ch/njol/skript/doc/HTMLGenerator.java

```diff
@@ -389,7 +389,7 @@ private String generateAnnotated(String descTemp, SyntaxElementInfo<?> info) {
 			String[] eventLinks = new String[eventNames.length];
 			for (int i = 0; i < eventNames.length; i++) {
 				String eventName = eventNames[i];
-				eventLinks[i] = "<a href=\"events.html#" + eventName + "\">" + eventName + "</a>";
+				eventLinks[i] = "<a href=\"events.html#" + eventName.replaceAll(" ?/ ?", "_").replaceAll(" +", "_") + "\">" + eventName + "</a>";
```

Comment:
can we just use URL encoding here instead?

Reply:
If I understand URL Encoding correctly If we use it then it won't work due to how the actual event link looks like, here is an example `https://skriptlang.github.io/Skript/events.html#bucket_fill` the sapces are replaced with `_` rather than `%20`

So the easier way to handle it is to just replace the spaces with an underscore `_` and because we have this event name `Break / Mine` I replaced `[ ]/[ ]` with an underscore as well.

Feel free to give me any info that I don't know about URL encoding if it easier and faster to handle than this

Reply:
I can also shorten this by using `.replaceAll("( ?/ ?| +)", "_")`